pkg/k8sclient/client.go

```diff
@@ -77,8 +97,8 @@ func GetResourceClient(apiVersion, kind, namespace string) (dynamic.ResourceInte
 }
 
 // GetKubeClient returns the kubernetes client used to create the dynamic client
-func GetKubeClient() kubernetes.Interface {
-	return kubeClient
+func (c *resourceClientFactory) GetKubeClient() kubernetes.Interface {
```

Comment:
This needs to remain a global function. We had to expose the k8s clientset upon multiple requests #190 #295 
So this should be:
```Go
func GetKubeClient() kubernetes.Interface {
  once.Do(newSingletonFactory)
  return singletonFactory.kubeClient
}
```